projects/pgai/pgai/cli.py

```diff
@@ -378,14 +388,24 @@ async def async_run_vectorizer_worker(
                 err_msg = f"unable to connect to database: {str(e)}"
             else:
                 err_msg = f"unexpected error: {str(e)}"
-            await handle_error(err_msg, None, worker_tracking, exit_on_error)
+            await handle_error(err_msg, vectorizer_id, worker_tracking, exit_on_error)
+        except BaseExceptionGroup as e:  # type: ignore
+            # catch any exceptions, log them, and keep on going
+            for exception in e.exceptions:  # type: ignore
+                for exception_line in traceback.format_exception(e):  # type: ignore
```

Comment:
```suggestion
                for exception_line in traceback.format_exception(exception):  # type: ignore
```

Reply:
Wait, I'll double check this, because I think I'm running python 3.10 and in the builtin version the traceback (I think) shows. the subexceptions

Reply:
Take a look at https://github.com/timescale/pgai/pull/597#discussion_r2022926476